File: be/src/main/java/org/example/gogoma/controller/response/ApplyResponse.java
package org.example.gogoma.controller.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import org.example.gogoma.domain.user.enums.Gender;

@Data
@AllArgsConstructor
public class ApplyResponse {

    private String name;
    private String email;
    private Gender gender;
    private String birthMonth;
    private String birthDay;
    private String birthYear;
    private String phoneNumber;
    private String roadAddress;
    private String detailAddress;
    private String clothingSize;

}

File: app/src/main/java/com/example/gogoma/data/model/ApplyResponse.kt
package com.example.gogoma.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class ApplyResponse(
    val name: String,
    val email: String,
    val gender: String,
    val birthMonth: String,
    val birthDay: String,
    val birthYear: String,
    val phoneNumber: String,
    val roadAddress: String,
    val detailAddress: String,
    val clothingSize: String
) : Parcelable
